django_sorcery/db/strategy_options.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
maybe add docstring that this is heavily inspired by https://docs.sqlalchemy.org/en/latest/_modules/examples/dogpile_caching/caching_query.html?